docs/references/backend/authenticate-request.mdx

```diff
@@ -22,36 +22,63 @@ function authenticateRequest: (request: Request, options: AuthenticateRequestOpt
 
 ## `AuthenticateRequestOptions`
 
+It is recommended to set these options as [environment variables](/docs/deployments/clerk-environment-variables#api-and-sdk-configuration) where possible, and then pass them to the function. For example, you can set the `secretKey` option using the `CLERK_SECRET_KEY` environment variable, and then pass it to the function like this: `createClerkClient({ secretKey: process.env.CLERK_SECRET_KEY })`.
```

Comment:
I think we should also mention the publishableKey here.

Reply:
I mention it throughout the doc - did you see the example sections?